### PR TITLE
Create EcoSynthesisX.yaml

### PR DESCRIPTION
**Pull request by PashaBurg**
Approving request

### CODE REVIEW

File: data/projects/e/EcoSynthesisX.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+version: 5
+name: EcoSynthesisX
+display_name: EcoSynthesisX
+github:
+- url: https://github.com/EcoSynthesisX
+npm:
+- url: https://www.npmjs.com/package/oss-directory
```
Comment (ryscheng) [inline data/projects/e/EcoSynthesisX.yaml:7]: This should only be used to enumerate your own npm packages. Please remove `oss-directory`

Comment (PashaBurg) [reply data/projects/e/EcoSynthesisX.yaml]: My bad. Deleted

Comment (PashaBurg) [reply data/projects/e/EcoSynthesisX.yaml]: Hello! Sorry, I'm not really deep in code. Did I do proper correction for the file? Are we on Open Source Observer?

Comment (ryscheng) [reply data/projects/e/EcoSynthesisX.yaml]: Once we merge this in, it should start indexing your project. Let me look at this now. Thanks!

**Comment by ryscheng** [comment]
@PashaBurg I think the YAML isn't properly formatted, check these logs:
https://github.com/opensource-observer/oss-directory/actions/runs/9279629186/job/25606642487?pr=307

The best way to check locally is to run 

```bash
pnpm install
pnpm validate
```

Can you help fix these errors first?

**Comment by PashaBurg** [comment]
Corrected .yaml done ✅

**Comment by PashaBurg** [comment]
@ryscheng Hello! Can you support on the next step? Should I initiate process somehow?

**Comment by ccerv1** [comment]
hey @PashaBurg we already have your project in OSS Directory 🎉 
I've added the missing deployer from your PR in my [new PR](https://github.com/opensource-observer/oss-directory/pull/323). I'll close at your PR and we should be good to go!

Here's the [latest version](https://github.com/opensource-observer/oss-directory/blob/main/data/projects/e/ecosynthesisx.yaml). If you see any issues, please open a new PR. Thanks again!

**Pull request closed without merging**